x-pack/plugins/cases/public/components/header_page/index.tsx

```diff
@@ -73,6 +74,7 @@ const HeaderPageComponent: React.FC<HeaderPageProps> = ({
   subtitle2,
   title,
   titleNode,
+  'data-test-subj': dataTestSubj,
```

Comment:
nit: we could rename `data-test-subj` to `dataTestSubj`

Reply:
I had to make this change to make it work properly. It is strange but it had to be done.